data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report10_extra.py

```diff
@@ -19,649 +19,594 @@
 # https://precisionmedicineinitiative.atlassian.net/browse/DC-1404
 #
 #
+# -
 
-# + papermill={"duration": 0.709639, "end_time": "2021-02-02T22:30:32.661373", "exception": false, "start_time": "2021-02-02T22:30:31.951734", "status": "completed"} tags=[]
 import urllib
 import pandas as pd
-
-from common import PIPELINE_TABLES
+from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
 pd.options.display.max_rows = 120
 
 # + papermill={"duration": 0.023643, "end_time": "2021-02-02T22:30:31.880820", "exception": false, "start_time": "2021-02-02T22:30:31.857177", "status": "completed"} tags=["parameters"]
 project_id = ""
-com_cdr = ""
-deid_cdr = ""
-ct_deid = ""
-ct_deid_sand = ""
-deid_sand = ""
-pipeline = ""
-run_as = ""
-# -
+rt_cdr_deid = ""
+ct_cdr_deid=""
+deid_sand=""
+pipeline=""
+rt_cdr_deid_clean=''
+reg_combine=''
+combine=''
+run_as=""
+cdr_cutoff_date=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns=['query', 'result'])
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
+#reg_combine = "reg_2022q2r4_combined"
```

Comment:
These parameters should be removed,  we do not want our dataset and project names in the git history.